django/publicmapping/static/js/mapping.js

```diff
@@ -137,7 +137,7 @@ function init() {
 
     // default map_server is on same host unless otherwise specified 
     if (MAP_SERVER === "") {
-        MAP_SERVER = window.location.host;
+        MAP_SERVER = "geoserver.internal.districtbuilder.com:8080"
```

Comment:
This wouldn't resolve outside of the VM, right?

Reply:
I was under the impression that, running in the container, it would have access to the container's networking. Is that not the case?

Reply:
But isn't `MAP_SERVER` being resolved in the browser? This seems similar to how we handle `TILE_SERVER_LOCATION ` inside of the RF UI.

Reply:
...yeah. Ok yeah :man_facepalming: 

i think i thought something like:

"it's in django so it's on the server this is fine"

:man_facepalming: